styles/16colors.json

```diff
@@ -0,0 +1,204 @@
+{
+  "document": {
+    "block_prefix": "\n",
+    "block_suffix": "\n",
+    "color": "7",
+    "margin": 2
+  },
+  "block_quote": {
+    "indent": 0,
+    "color": "3",
+    "indent_token": "│ ",
+    "italic": true,
+    "margin": 1
+  },
+  "paragraph": {},
+  "list": {
+    "level_indent": 2,
+    "color": "7"
+  },
+  "heading": {
+    "block_suffix": "\n",
+    "color": "4",
```

Comment:
As said earlier, with my theme, 3 looks better. I found that setting 3 to a bright color looks better in some tools (htop, eza for instance), and in turns this make it more suitable to headings in my opinion. Very subjective though :-)